workers/tests/integrationtests/indicators/test_mapping_saturation.py

```diff
@@ -0,0 +1,147 @@
+import asyncio
+from datetime import datetime
+
+import numpy as np
+import pytest
+
+from ohsome_quality_analyst.indicators.mapping_saturation.indicator import (
+    MappingSaturation,
+)
+from tests.integrationtests.utils import oqt_vcr
+
+
+class TestCheckEdgeCases:
+    @pytest.fixture()
+    def indicator(self, topic_building_count, feature_germany_heidelberg):
+        i = MappingSaturation(topic_building_count, feature_germany_heidelberg)
+        return i
+
+    def test_no_data(self, indicator):
+        indicator.values = [0, 0]
+        assert indicator.check_edge_cases() == "No features were mapped in this region."
+
+    def test_not_enough_data(self, indicator):
+        indicator.values = [0, 1]
+        assert (
+            indicator.check_edge_cases()
+            == "Not enough data in total available in this region."
+        )
+
+    def test_not_enough_data_points(self, indicator):
+        indicator.values = [0, 1, 10]
+        assert indicator.check_edge_cases() == (
+            "Not enough data points available in this regions. "
+            + "The Mapping Saturation indicator needs data for at least 36 months."
+        )
+
+    def test_data_deleted(self, indicator):
+        indicator.values = [i for i in range(36)]
+        indicator.latest_value = 0
```

Comment:
You could remove the `latest_value` implementation in favour of `values[-1]`

Reply:
Done